tripleblindmarket/private-kit
# Add accessibility labels to images and svgs

#### Description:

This commit adds the accessibility tag and accessibility labels to images and icons throughout the BT and GPS app.

Note: These changes did not include self-assessment as that design may change and we will not be launching for MVP. We will also need to add translations for languages other than English.

#### Linked issues:

https://trello.com/c/kvxfTxo2/192-as-a-user-in-minnesota-using-a-mobile-screen-reader-i-need-to-ensure-that-all-non-text-content-has-available-and-accurate-text-a

#### Screenshots:

No changes to the UI

#### How to test:

- Run app locally on a screen reader
- Ensure all images and icons are provided with a clear, concise, and accurate description
